src/syn/io/der.rs

```diff
@@ -145,7 +155,34 @@ impl Reader for DerReader {
         &mut self,
         f: F,
     ) -> Result<S, Self::Error> {
-        unimplemented!()
+        let (class, pc, tag) = self.buffer.read_identifier()?;
+        let length = self.buffer.read_length()?;
+
```

Comment:
I don't understand why we are reading here in a `write_enumerated` function.
Since cfb21015ccb255e7032b2dc1eca5f845e88ffe1b `C::Tag` provides the `Tag` of the ENUMERATED instance.

Reply:
It's not actually in `write_enumerated`, Github collapsed some lines here

Reply:
Sorry, my bad. You are right